src/fu-util.c

```diff
@@ -4094,6 +4096,14 @@ main(int argc, char *argv[])
 					 /* TRANSLATORS: command line option */
 					 N_("Output in JSON format"),
 					 NULL},
+					{"authenticate",
+					 '\0',
+					 0,
+					 G_OPTION_ARG_NONE,
+					 &priv->authenticate,
```

Comment:
I think this would be more natural the other way around, e.g. authenticate by default, but "opt out" for the case where you don't want the polkit auth.

Reply:
swapped it around